### ISSUE TITLE
Save yielded OOIs even when normalizer crashes

### ISSUE THREAD

**Issue by ammar92** [minvws/nl-kat-coordination#752]
Currently, a normalizer is run and results are only saved if the normalizer executes normally. This way, when a normalizer crashes at a later point (due to e.g. programming errors or invalid user input), potentially useful yielded OOIs might get lost because we don't save them.

**Describe the solution you'd like**
I would like to save every yielded OOI from the normalizer.

**Comment by Donnype** [comment]
Are you referring to crashes within the normalizer itself? Because I can imagine yielding them one by one might force us to validate consistency with foreign key references ourselves now that we do not need Octopoes to return OOIs anymore. Perhaps this implicit ordering comes naturally writing the normalizers, but it does open up a window for potential inconsistencies without proper checking.

Do we encounter the issue often?

**Comment by ammar92** [comment]
> Are you referring to crashes within the normalizer itself? Because I can imagine yielding them one by one might force us to validate consistency with foreign key references ourselves now that we do not need Octopoes to return OOIs anymore. Perhaps this implicit ordering comes naturally writing the normalizers, but it does open up a window for potential inconsistencies without proper checking.

Yes, I am referring to the crashes within the normalizer itself. And I'm not talking about yielding the results one by one, but what I mean is gather the OOI results yielded by the `run` function/ generator and save them regardless of wether an exception occurs within the function/ generator or not.

> Do we encounter the issue often?

It doesn't happen often, but it happens
